shared/wallets/send-form/choose-asset/index.js

```diff
@@ -204,31 +189,30 @@ type DisplayChoiceProps = {
   symbol: string,
 }
 const DisplayChoice = (props: DisplayChoiceProps) => (
-  <ClickableBox hoverColor={globalColors.blue4} onClick={props.onClick}>
-    <Box2
-      direction="horizontal"
-      style={styles.choiceContainer}
-      fullWidth={true}
-      gap="small"
-      gapStart={true}
-      gapEnd={true}
-    >
-      <Text type="Body" style={props.selected ? styles.blue : undefined}>
-        {props.symbol === 'XLM' ? 'Purely strictly ' : 'Lumens (XLM) displayed as '}
-        <Text type="BodyExtrabold" style={props.selected ? styles.blue : undefined}>
-          {props.currencyCode} ({props.symbol})
-        </Text>
-      </Text>
-      <Box2 direction="horizontal" style={styles.spacer} />
-      {props.selected && (
-        <Icon
-          type="iconfont-check"
-          color={globalColors.blue}
-          style={iconCastPlatformStyles(styles.checkIcon)}
-        />
-      )}
-    </Box2>
-  </ClickableBox>
+  <Kb.ClickableBox
+    hoverColor={Styles.globalColors.blue4}
+    onClick={props.onClick}
+    style={styles.displayChoice}
+  >
+    <Kb.Box2 direction="horizontal" style={styles.choiceContainer} fullWidth={true}>
+      <Kb.Text
+        type="Body"
+        style={Styles.collapseStyles([styles.choice, props.selected ? styles.blue : undefined])}
+      >
+        {props.symbol === 'XLM' ? 'Purely strictly ' : 'Lumens displayed as '}
```

Comment:
remove `(XLM)` which matches design and reduces the width which is too long for the 5s